### PR TITLE
draft blog for RSE associations & finances blog

### PR DESCRIPTION
**Pull request by danielskatz**
This has the content from the previous google doc, and has authors listed as people who contributed to that doc directly. I might have missed authors, so please feel free to add yourself if you contributed.  And if there's anything for your org that you want to change, please change it.

### CODE REVIEW

File: _posts/2021-04-02-RSE-financial-organizations.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+- Ben van Werkhoven
+- Frank Löffler
+
+This blog post is intended to talk about how different RSE associations
+financially function (purely volunteer; income from membership, events,
+sponsors, etc.; costs for events, staff, other). While it is now being
+posted as a blog, it's really a snapshot in time, as we expect the
+RSE associations to change their financial models over time. This post
+may be useful to the existing organizations that want to consider
+changes, and to new organization that need to make financial decisions.
```
Comment (sdruskat) [inline _posts/2021-04-02-RSE-financial-organizations.md:21]:
```suggestion
changes, and to new organizations that need to make financial decisions.
```

Fix typo.

File: _posts/2021-04-02-RSE-financial-organizations.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+* be-RSE: The Belgian Research Software Association is not a formal organisation yet, 
+nor is it financially independant.
```
Comment (sdruskat) [inline _posts/2021-04-02-RSE-financial-organizations.md:24]:
```suggestion
nor is it financially independent.
```

Fix typo.

File: _posts/2021-04-02-RSE-financial-organizations.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+All efforts is driven by volunteers of various organisations. At the moment, two organisations,
```
Comment (sdruskat) [inline _posts/2021-04-02-RSE-financial-organizations.md:25]:
```suggestion
All efforts are driven by volunteers of various organisations. At the moment, two organisations, 
```
Fix typo.

File: _posts/2021-04-02-RSE-financial-organizations.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+may be useful to the existing organizations that want to consider
+changes, and to new organization that need to make financial decisions.
+
+* be-RSE: The Belgian Research Software Association is not a formal organisation yet, 
+nor is it financially independant.
+All efforts is driven by volunteers of various organisations. At the moment, two organisations, 
+VIB and KU Leuven are driving the activities.
+ 
+* de-RSE: membership fees (~60 €/y): not that much, plan to support
+local groups and national conference
```
Comment (sdruskat) [inline _posts/2021-04-02-RSE-financial-organizations.md:29]:
```suggestion
* de-RSE: financed through membership fees (€60/a for full, €30/a for discounted membership), used exclusively to support chartered aims, e.g., local groups, national conference, etc.
```

Restructure de-RSE section

File: _posts/2021-04-02-RSE-financial-organizations.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+* Nordic-RSE: not financial as yet but funded by a “Nordic project”.
```
Comment (sdruskat) [inline _posts/2021-04-02-RSE-financial-organizations.md:31]:
```suggestion
* Nordic-RSE: no own finances as of yet, funded by a "Nordic project".
```

Have I understood this correctly? Tbc by @bast/@annefou.

File: _posts/2021-04-02-RSE-financial-organizations.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+image: /img/financial-org.png
+---
+Photo by [Micheile Henderson](https://unsplash.com/@micheile) on [Unsplash](https://unsplash.com/).
+  
+**Authors**
+
+- Daniel S. Katz
+- Anne Claire Fouilloux
+- Ben van Werkhoven
+- Frank Löffler
```
Comment (sdruskat) [inline _posts/2021-04-02-RSE-financial-organizations.md:13]:
```suggestion
- Frank Löffler
- Stephan Druskat
```

**Comment by sdruskat** [comment]
Good to go from my side once @bast or @annefou confirm the changes I've made to the Nordic section, trying to make it more understandable (to me).

File: _posts/2021-04-02-RSE-financial-organizations.md (hunk 4 above)
Comment (danielskatz) [reply _posts/2021-04-02-RSE-financial-organizations.md:30]:
```suggestion
* de-RSE: Formal association with charitable status (includes tax exemptions), financed through membership fees (€60/year for full membership; €30/year for discounted membership), used exclusively to support chartered aims, e.g., local groups, national conference, etc. Financial support for deRSE19 came from corporate and charitable sponsors and the Gesellschaft für Informatik. Society uses academic and commercial infrastructure for work (GWDG's chat and cloud, GitHub).
```

File: _posts/2021-04-02-RSE-financial-organizations.md (hunk 5 above)
Comment (bast) [reply _posts/2021-04-02-RSE-financial-organizations.md:31]: Sorry for late follow-up. I would change the sentence after the comma to "All efforts are driven by volunteers of various organisations". (wording taken from this text, above)

I think this is more accurate. The financing of the Nordic RSE conference is on the CodeRefinery/NeIC budget but so far has not been used so the efforts have in reality been volunteer efforts and in-kind efforts.

File: _posts/2021-04-02-RSE-financial-organizations.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+changes, and to new organizations that need to make financial decisions.
+
+* be-RSE: The Belgian Research Software Association is not a formal organisation yet, 
+nor is it financially independent.
+All efforts are driven by volunteers of various organisations. At the moment, two organisations, 
+VIB and KU Leuven are driving the activities.
+ 
+* de-RSE: Formal association with charitable status (includes tax exemptions), financed through membership fees (€60/year for full membership; €30/year for discounted membership), used exclusively to support chartered aims, e.g., local groups, national conference, etc. Financial support for deRSE19 came from corporate and charitable sponsors and the Gesellschaft für Informatik. Society uses academic and commercial infrastructure for work (GWDG's chat and cloud, GitHub).
+
+* Nordic-RSE: not financial as yet but funded by a “Nordic project”.
```
Comment (danielskatz) [inline _posts/2021-04-02-RSE-financial-organizations.md:31]:
```suggestion
* Nordic-RSE: Does not have its own finances as of yet. All efforts are driven by volunteers for various organizations.
```

Comment (danielskatz) [reply _posts/2021-04-02-RSE-financial-organizations.md:31]: @bast - is this ok?

Comment (danielskatz) [inline _posts/2021-04-02-RSE-financial-organizations.md:31]:
```suggestion
* Nordic-RSE: Does not have its own finances as of yet. All efforts are driven by volunteers from various organizations.
```

Comment (bast) [reply _posts/2021-04-02-RSE-financial-organizations.md:31]: Yes. Thank you!

Comment (danielskatz) [reply _posts/2021-04-02-RSE-financial-organizations.md:31]: @bast, is this ok?

**Review by bast — APPROVED** [review]
Thank you for your work on this!

**Comment by abotzki** [comment]
Thanks a lot - approved on behalf of be-RSE.

**Comment by knarrff** [comment]
Thanks a lot. I have two questions:
- (How) Do we need to give credit to the image / does it have a license?
- Would it be ok to reduce the image size from 13MB to something that still looks good but is a little smaller?

**Comment by danielskatz** [comment]
> Thanks a lot. I have two questions:
>
> (How) Do we need to give credit to the image / does it have a license?

line 6 of the blog post source gives credit for the image.

> Would it be ok to reduce the image size from 13MB to something that still looks good but is a little smaller?

sure, though I don't think it's needed

**Comment by knarrff** [comment]
Ah, missed the credit. I'll try to tinker with the image. 13MB is quite some size for someone with a thin connection.

**Comment by knarrff** [comment]
I've created a pull-request against your fork.

**Comment by danielskatz** [comment]
thanks - merged

**Comment by knarrff** [comment]
Thanks. Approved by de-RSE.

**Comment by sdruskat** [comment]
@danielskatz Can you please @ ping me in this discussion to let me know when this can be merged? Thanks!

**Comment by danielskatz** [comment]
👋 @nrmay - Is this ok with you to publish?

**Review by nrmay — APPROVED** [review]
Ok from RSE-AUNZ perspective.

**Comment by nrmay** [comment]
Approved. 
-------- Original message --------From: "Daniel S. Katz" ***@***.***> Date: 27/4/21  22:07  (GMT+10:00) To: "RSE-leaders/researchsoftware.org" ***@***.***> Cc: Nick May ***@***.***>, Mention ***@***.***> Subject: Re: [RSE-leaders/researchsoftware.org] draft blog for RSE associations & finances blog (#35) 

👋 @nrmay - Is this ok with you to publish?

**Comment by danielskatz** [comment]
👋 @sdruskat - this is ready to publish now.

**Comment by sdruskat** [comment]
Merged, thanks @danielskatz and all!
I've included a break marker to show some teaser text on the blog page.

**Comment by sdruskat** [comment]
@all-contributors please add @danielskatz, @annefou, @benvanwerkhoven, @knarrff for blog and review

**Comment by sdruskat** [comment]
@all-contributors please add @danielskatz for blog and review

**Comment by sdruskat** [comment]
@all-contributors add @annefou for blog and review

**Comment by sdruskat** [comment]
@all-contributors add @benvanwerkhoven for blog, review and maintenance

**Comment by sdruskat** [comment]
@all-contributors add @knarrff for blog and review

**Comment by sdruskat** [comment]
@all-contributors add @bast for review

**Comment by sdruskat** [comment]
@all-contributors add @nrmay for review

**Comment by sdruskat** [comment]
@all-contributors add @abotzki for review

**Pull request merged**